Enterprise-CMCS/macpro-mdct-mcr
# MDCT-2211: Add report type selector to Admin dashboard

## Summary

### Description

Adding a radio selection field to the Admin dashboard on MCR for admin users to be able to navigate to the correct report type (MCPAR or MLR).

### Related ticket

[MDCT-2211](https://qmacbis.atlassian.net/browse/MDCT-2211)

### How to test

- Log in as `adminuser@test.com`
- Select a state and a report type

Make sure that:
- `Go to Report Dashboard` button is disabled until you pick a report type
- Cannot navigate to the report until you pick a state
- Navigate to the corresponding report path

### Important updates

N/A

### Author checklist
